catalogue/webapp/components/SearchForm/SearchForm.js

```diff
@@ -40,14 +41,21 @@ const ClearSearch = styled.button`
 `;
 
 const SearchForm = ({ ariaDescribedBy, compact }: Props) => {
-  const { query, workType, _queryType, setQueryType } = useContext(
-    CatalogueSearchContext
-  );
+  const {
+    query,
+    workType,
+    _queryType,
+    setQueryType,
+    dateFrom,
```

Comment:
Could we keep these as `_dateFrom` similar to `_queryType`

Reply:
Sure thing. Is the reason to keep them underscored just for the sake of transparency (that what's in the URL is the same as what's being asked of the API), or something else?

Reply:
It denotes them being test variables, and are probably not the final shape of the API - so letting other clients know not to rely on them.

Reply:
Got it. Makes sense.